src/machine/machine_mimxrt1062_i2c.go

```diff
@@ -212,13 +207,13 @@ func (i2c I2C) Tx(addr uint16, w, r []byte) error {
 	return nil
 }
 
-// WriteRegister transmits first the register and then the data to the
+// WriteRegisterEx transmits first the register and then the data to the
 // peripheral device.
 //
 // Many I2C-compatible devices are organized in terms of registers. This method
 // is a shortcut to easily write to such registers. Also, it only works for
 // devices with 7-bit addresses, which is the vast majority.
-func (i2c I2C) WriteRegister(address uint8, register uint8, data []byte) error {
+func (i2c I2C) WriteRegisterEx(address uint8, register uint8, data []byte) error {
```

Comment:
Why `WriteRegisterEx` and `ReadRegisterEx`?
If it's to avoid a duplicate method, I think this is a better solution: https://github.com/tinygo-org/tinygo/pull/4130